### ISSUE TITLE
Login to CLS asking for environmental variables

### ISSUE THREAD

**Issue by luisexperian** [Nuix/AutomatedNuixUCRT#1]
We are setting the username and password in the settings.json file (not secure, used just for testing) and it keeps failing because:

2023-05-11 15:01:25.506 -0500 [main] 153961 ERROR com.nuix.util.auth.d - Username/password obtained via environment variables do not authenticate

How we are supposed to setup the login to CLS?

**Comment by JuicyDragon** [comment]
Hello @luisexperian 

I believe the CLS credentials are being resolved from environment variables `NUIX_USERNAME` and `NUIX_PASSWORD` as that is where Nuix should be pulling them from.

I did a quick search through the code and did not find anywhere that those are being set by the script.  I believe the [other version](https://github.com/Nuix/NuixUCRT) of this script includes a .NET application which would use the values from the **settings.json** files to inject those environment variables into the Nuix process it starts.  Since this version does not have the .NET application, it must be using the actual environment variables and not temporarily overridden values.

Does that make sense?

**Comment by luisexperian** [comment]
Makes sense and after adding those to the windows env variables, it fails saying the variables already exist and can not be added.
